api/types/plugin.go

```diff
@@ -183,6 +185,16 @@ func (p *PluginV1) CheckAndSetDefaults() error {
 		if len(staticCreds.Labels) == 0 {
 			return trace.BadParameter("labels must be specified")
 		}
+
+	case *PluginSpecV1_Jira:
+		if settings.Jira == nil {
+			return trace.BadParameter("missing Jira settings")
+		}
+
+		if err := settings.Jira.CheckAndSetDefaults(); err != nil {
+			return trace.Wrap(err)
+		}
```

Comment:
does it require static credential? if yes, you probably also need to check `staticCreds := p.Credentials.GetStaticCredentialsRef()`?